src/pypi_attestations/_impl.py

```diff
@@ -388,6 +478,15 @@ class GitHubPublisher(_PublisherBase):
     action was performed from.
     """
 
+    def _as_policy(self) -> VerificationPolicy:
+        policies: list[VerificationPolicy] = [
+            policy.OIDCIssuerV2("https://token.actions.githubusercontent.com"),
+            policy.OIDCSourceRepositoryURI(f"https://github.com/{self.repository}"),
+            _GitHubTrustedPublisherPolicy(self.repository, self.workflow),
```

Comment:
nit: the name `_GitHubTrustedPublisherPolicy` suggests that it's, by itself, the full policy for GitHub. Maybe we could rename it to something like `_GitHubBuildConfigURIPolicy`?

Or alternatively, make `_GitHubTrustedPublisherPolicy` return the full `AllOf(OIDCIssuer,OIDCSourceRep,...)` policy?

Reply:
Ah yeah, good idea -- I'll make it contain the `AllOf` internally.